File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        
//         int red = 0;
//         int white = 0;
//         int blue = 0;
        
//         for(int i=0; i<nums.length; i++){
//            if(nums[i] == 0){
//                red++;
//            }else if(nums[i]==1){
//                white++;
//            }else{
//                blue++;
//            }
//         }
        
       for(int i=0; i<nums.length-1; i++){
           for(int j=i+1; j<nums.length; j++){
               if(nums[j] < nums[i]){
                   int temp = nums[i];
                   nums[i] = nums[j];
                   nums[j] = temp;
               }
           }
       }
    
    }
}

File: 0075-sort-colors/0075-sort-colors.ts
/**
 Do not return anything, modify nums in-place instead.
 */
function sortColors(nums: number[]): void {
    
    let noSwaps = true;
    for(let i=nums.length; i>0; i--){
        for(let j=0; j<i-1; j++){
            if(nums[j] > nums[j+1]){
                noSwaps = false;
                let temp = nums[j];
                nums[j] = nums[j+1];
                nums[j+1] = temp;
            }
        }
          if(noSwaps) break;
    }

};